modules/cli/src/main/kotlin/datamaintain/cli/documentation/RebuildDocumentation.kt

```diff
@@ -0,0 +1,33 @@
+package datamaintain.cli.documentation
+
+import com.github.ajalt.clikt.core.Context
+import com.github.ajalt.clikt.core.PrintHelpMessage
+import datamaintain.cli.app.datamaintainApp
+import java.io.File
+
+fun main() {
+    File("../../docs/cli-configuration.md").writeText(buildDocumentation())
+}
+
+fun buildDocumentation(): String {
```

Comment:
I guess it is not a problem here on Datamaintain because the documentation will be small, but we could have write on an IOStream rather than building a String in memory then written on a file.